README.md

```diff
@@ -47,12 +47,18 @@ Since there are many images, they are in different stages of their life cycle. T
 
 ## Development
 
-Images can be built and run locally with this repository. This has the following requirements:
+**Images can be built and run locally with this repository. This has the following requirements:**
 
 - Local System of Linux (Ubuntu tested) or macOS
 - Bash (v4+)
 - Docker Engine (v19.03+)
 
+**Images can also be built using the Harness CI Pipeline (Reference Pipeline)**
+
+- Step 1: Fork the [repository](https://github.com/codewdhruv/sample-docker-image-release) & use import from git feature to fetch the ```pipelinereleasesample.yaml``` in Harness App
```

Comment:
I see you referenced your repo! does this need a new home in Harness Community?

Reply:
this is under review. we can add this in harness-community after the testing is over.